### DISCUSSION TITLE
Suggestions regarding Wareffort, AQ Release, r14 gear etc.

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#3858] [Suggestions]
Hi,
some suggestions regarding the next phase:

**War Effort**

1. I personally think this is an opportunity to "remove" some of the excess economies we have currently, which would would be a shame to skip. The amounts would still be need some tuning, but having it accessible, would be good in my opinion.

**AQ releases**
1. going hand in hand with the war effort, I suggest an gated release of the raids. Essentially, on day zero, the quests and npcs became available for most if not all stuff and war effort collections and the GONG quest chain can be started. But, AQ20 and AQ40 are closed in the beginning. AQ20 could unlock once a certain (lower) threshold of the war effort is fulfilled, while AQ40 would require a higher threshold. In both cases, the raid would release on the next wednesday, after the threshold has been reached to enable those looking for a race, to race.
2. the opening event (in case you wanna script that, too) might not be fully integrated into the first or second release tho.

**Gong Mount**
1. I am personally against making the mount available as there are players already progressed on the quests / having some reputation ahead of others and the likelihood of some kind of bug appearing and either too many or actually not qualified people getting the mount and causing drama seems rather high to me.

**PvP**
1. to make this a PvE and PvP combined effort, I would suggest that we use the currently implemented battleground tokens, and require them also as part of the war effort turn ins. This way, pvp and pve players would be required to put in some work.
2. this would also allow e.g to do release e.g. half the pvp set with aq20 and half the pvp set with aq40. for the weapons, I would still postpone those a bit further personally or add further requirements.

**Comment by Elexias** [comment]
Hello everyone,
I'm very excited about new announcements and progression on AQ phase of server, this was kinda faster than I've expected, since ZG was released in middle of July, I've been playing here for very long and some phases were so time consuming while this one was super fast, which I like!

I have already suggested many things regarding AQ phase, but those were usually on discord chat, never officially as discussion post. I will leave my though here

**AQ20 & AQ40 Release**
I like your idea of opening the gates unofficially so we can enter AQ20 while not being able to enter AQ40 before War Effort and AQ chain is done on server.

**Opening the Gates of AQ**
While you announced nice things about AQ20 and AQ40, there is minimum or let's not say zero information about questline itself, quest line has been bounced a lot on WotLK, there is always some changes on different servers, so my question is:

- How many people would be able to loot Broodlords head? One or whole raid, but please make it official answer
- Will War Effort event and Opening last for 7 days?
- Will first part of quest line, bug killing stay like it used to be on classic, lot of farm etc?

**War Effort**
I wanna hear Staff though on War effort mats requirement, server is pretty stable and good, lot of past banned players had tons of mats already prepared, some of them are maybe still on some of not banned accounts or maybe handled to other people, this is nothing forbidden, people started gathering mats on time. Mats required were insanely high, but on population like this, personally I would like to see around 20-30% requirements of original. If it is only one drop per raid for starting quest for Opening the gates, and you put low amount of requirements around 5% for example, quest line will be finished within 2 weeks and people will gather mats fast enough to have only 10 people on server doing whole quest line. I'm not sure what is your idea as staff, do you have 50% of server to have it or to have 10% of people on server who worked hard or idk. Just wanna make sure that I'm rushing into the right thing!

**Comment by Elexias** [comment]
> 1. I am personally against making the mount available as there are players already progressed on the quests / having some reputation ahead of others and the likelihood of some kind of bug appearing and either too many or actually not qualified people getting the mount and causing drama seems rather high to me.

What do you mean with bug appearing? Who has reputation, name them? All bug exploiters were banned on this server very fast. Which part of quests are progressed and who has it? I for example have mats prepared, but those are the ones that you can farm ahead.

**Reply by balleny** [reply]
we have had plenty of implementations, also custom implementations, that got released in a buggy state and have been fixed only 1-2 weeks after quite a few people took advantage of these issues.
I kinda expect the same stuff to happen here.
